awscli/customizations/cloudfront.py

```diff
@@ -58,6 +70,44 @@ def add_to_params(self, parameters, value):
                 }
 
 
+def _add_default_root_object(argument_table, **kwargs):
+    argument_table['distribution-config'].required = False
+    argument_table['default-root-object'] = DefaultRootObjectArgument(
+        kwargs['session'])
```

Comment:
You can just add session to the signature to retrieve it. You do not need to pull it from kwargs.